homeassistant/components/lamarzocco/manifest.json

```diff
@@ -0,0 +1,27 @@
+{
+  "domain": "lamarzocco",
+  "name": "La Marzocco",
+  "bluetooth": [
+    {
+      "local_name": "MICRA_*"
+    },
+    {
+      "local_name": "MINI_*"
+    },
+    {
+      "local_name": "GS3_*"
+    },
+    {
+      "local_name": "GS3AV_*"
+    }
+  ],
+  "codeowners": ["@rccoleman", "@zweckj"],
```

Comment:
@rccoleman Are you willing to be a code owner?

Reply:
removed Rob for the moment to avoid blocking this PR. Will gladly add you back if I hear from you!